bootstrap/sync_third_party.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 The Absolute Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+. bootstrap/common/platform_info.sh
+. bootstrap/common/util.sh
+
+function sync_node() {
+  local target_path="./third_party/node"
+  local base_url="https://nodejs.org/dist/v6.9.5"
+
+  case $(get_platform_name) in
+    windows_x86) local target_url="/node-v6.9.5-win-x86.zip" ;;
```

Comment:
I think it's unnecessary. because we can not sure the link is always available.
We should replace the hard coded values with configuration files or deps files.